plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointEntryIterator.java

```diff
@@ -627,6 +678,12 @@ private void fillNextEntries()
         }
     }
 
+    @VisibleForTesting
+    long getCompletedBytes()
```

Comment:
Can we use `getCompletedPositions` instead for testing ? It should give you the number of input rows in the parquet reader and should be a more stable number than the bytes read.

Reply:
This is much cleaner. We can now assert exactly how many positions will be read per type.